horreum-api/src/main/java/io/hyperfoil/tools/horreum/api/services/TestService.java

```diff
@@ -216,15 +217,25 @@ void updateNotifications(@PathParam("id") int id,
            @Parameter(name = "sort", description = "json path to sortable value or start or stop for sorting by time",example = "$.label or start or stop"),
            @Parameter(name = "direction",description = "either Ascending or Descending",example="count"),
            @Parameter(name = "limit",description = "the maximum number of results to include",example="10"),
-           @Parameter(name = "page",description = "which page to skip to when using a limit",example="2")
+           @Parameter(name = "page",description = "which page to skip to when using a limit",example="2"),
+           @Parameter(name = "include", description = "label name(s) to include in the result as scalar or comma separated",
+                   examples = {
+                           @ExampleObject(name="single", value="id", description = "including a single label"),
+                           @ExampleObject(name="multiple", value="id,count", description = "including multiple labels")
+                   }),
+           @Parameter(name = "exclude", description = "label name(s) to exclude from the result as scalar or comma separated",
+                   examples = {
+                           @ExampleObject(name="single", value="id", description = "excluding a single label"),
+                           @ExampleObject(name="multiple", value="id,count", description = "excluding multiple labels")
+                   })
    })
    @APIResponses(
            value = { @APIResponse( responseCode = "200",
                    content = {
                            @Content ( schema = @Schema(type = SchemaType.ARRAY, implementation = ExportedLabelValues.class)) }
            )}
    )
-   List<ExportedLabelValues> listLabelValues(
+   List<ExportedLabelValues> labelValues(
```

Comment:
We need to go over existing docs that reference the previous name and update them to the new.

Reply:
Did you see docs that referenced the previous name that are not in the PR? I did a search for the `listLabelValues` substring in the Horreum project when I changed the method.

Reply:
Ah, sorry, mb! I assumed you had changed the endpoint definition as well, no need to make any changes then!